.github/workflows/molecule.yml

```diff
@@ -3,9 +3,9 @@ name: Molecule Test
 on:
   push:
     tags:
-      - '*'
+      - "*"
     branches:
-      - main
+      - "*"
```

Comment:
That change is not neccessary. The actions should not run on every branch but they should run on pull requests. So to have checks, just push a branch and create a merge requests. Nobody forces you to push every single commit. You can always run your checks locally. But now and then you should push what you build. That way the data is safe from hardware failure on your workstations and the checks will run.